src/routes/update-pops-waste.test.js

```diff
@@ -0,0 +1,94 @@
+import { jest } from '@jest/globals'
+import { httpClients } from '../common/helpers/http-client.js'
+import { updatePopsWaste } from './update-pops-waste.js'
+import Boom from '@hapi/boom'
+
+// Mock the httpClients
+jest.mock('../common/helpers/http-client.js', () => ({
+  httpClients: {
+    wasteMovement: {
+      put: jest.fn()
+    }
+  }
+}))
+
+describe('updatePopsWaste route', () => {
```

Comment:
I am not sure we should be unit testing the spec here. I think it belongs to black box tests that @LewisMagri is working on.